### ISSUE TITLE
Documentation for 2.93, and above?

### ISSUE THREAD

**Issue by ThrillDaWill** [mrachinskiy/commotion#23]
Hello, I tried following your tutorials but it's hard to follow since they are using the old UI, a lot has changed and a new tutorial on how to use your plugin would really help out!

**Comment by mrachinskiy** [comment]
Hi, Commotion does require full documentation not just video tutorials, but unfortunately I do not have time for it.
It will be done eventually, but not now.
